shopinvader_payment/__manifest__.py

```diff
@@ -5,19 +5,16 @@
 {
     "name": "Shopinvader Payment",
     "summary": "Payment integration for Shopinvader",
-    "version": "12.0.2.0.1",
+    "version": "13.0.1.0.0",
     "category": "e-commerce",
-    "website": "https://akretion.com",
+    "website": "https://github.com/shopinvader/odoo-shopinvader-payment",
     "author": "Akretion",
     "license": "AGPL-3",
-    "application": True,
-    "installable": False,
+    "installable": True,
     "external_dependencies": {"python": ["cerberus", "unidecode"], "bin": []},
     "depends": [
-        "account_payment_mode",
-        "account_payment_sale",
         "shopinvader",
-        "sale_automatic_workflow_payment_mode",
+        "sale_automatic_workflow_payment",
```

Comment:
Is this a hard dependency?

Reply:
@lmignon Your advice on this ? Maybe this could be avoided

Reply:
Removed

Reply:
it's no more a hard dependency :tada: 
It should be removed